scripts/travis/run_test.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
+    scripts/travis/osx/build.sh
+else
+    scripts/travis/linux/docker.sh rocproject/ci-ubuntu:17.04 scripts/travis/linux/ubuntu-17.04.sh
+    scripts/travis/linux/docker.sh rocproject/ci-ubuntu:16.04 scripts/travis/linux/ubuntu-16.04.sh
+    scripts/travis/linux/docker.sh rocproject/ci-ubuntu:14.04 scripts/travis/linux/ubuntu-14.04.sh
+    scripts/travis/linux/docker.sh rocproject/ci-fedora scripts/travis/linux/fedora.sh
+    scripts/travis/linux/docker.sh rocproject/ci-debian scripts/travis/linux/debian.sh
+    scripts/travis/linux/docker.sh rocproject/ci-centos scripts/travis/linux/centos.sh
+    scripts/travis/linux/docker.sh rocproject/ci-empty scripts/travis/linux/empty.sh
+    scripts/travis/linux/docker.sh rocproject/cross-raspberry scripts/travis/linux/raspberry.sh
+    scripts/travis/linux/docker.sh rocproject/cross-linaro scripts/travis/linux/linaro.sh
+    cat build.status
+fi
```

Comment:
If one of the linux scripts would fail, run_test.sh will still exit successfully and the build won't fail.

Reply:
Fixed